cirq-core/cirq/qis/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Tools and methods for quantum information science."""
+"""Tools and methods for primitives arising in quantum information science."""
```

Comment:
is "used in" OK here or are the tools and methods really "arising"?

Reply:
This is better.  Replaced.